LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MessageProcessor.cs

```diff
@@ -635,21 +635,27 @@ private async Task<DownlinkPktFwdMessage> ProcessJoinRequest(LoRaMessageWrapper
                 //in this case the second join windows must be used
                 if ((DateTime.UtcNow - startTimeProcessing) > TimeSpan.FromMilliseconds(RegionFactory.CurrentRegion.join_accept_delay1 * 1000 - 100))
                 {
-                    Logger.Log(devEui, $"processing of the join request took too long, using second join accept receive window", Logger.LoggingLevel.Info);
-                    tmst = loraMessage.PktFwdPayload.GetPktFwdMessage().Rxpks[0].tmst + RegionFactory.CurrentRegion.join_accept_delay2 * 1000000;
-                    if (string.IsNullOrEmpty(configuration.Rx2DataRate))
-                    {
-                        Logger.Log(devEui, $"using standard second receive windows for join request", Logger.LoggingLevel.Info);
-                        //using EU fix DR for RX2
-                        freq = RegionFactory.CurrentRegion.RX2DefaultReceiveWindows.frequency;
-                        datr = RegionFactory.CurrentRegion.DRtoConfiguration[RegionFactory.CurrentRegion.RX2DefaultReceiveWindows.dr].configuration;
+                    if((DateTime.UtcNow - startTimeProcessing).Seconds>=RegionFactory.CurrentRegion.join_accept_delay2){

```

Comment:
Can we have a PR without the changes on the core so only the SPI dev stuff. You can then PR this stuff separately